modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -35,10 +35,10 @@ def merge_clinics(appointments)
             cached_clinics[clinic_id] = get_clinic(appt[:location_id], clinic_id) unless cached
 
             service_name = cached_clinics.dig(clinic_id, :service_name)
-            appt[:service_name] = service_name if service_name
```

Comment:
I doubt it matters, but I believe these are functional changes. If there was no appt[:service_name] key, this will add it and set it to nil, which is slightly different than what VAOS is doing.